README.md

```diff
@@ -39,6 +39,9 @@ datDns.resolveName('foo.com', {noDnsOverHttps: true})
 // dont use .well-known/dat
 datDns.resolveName('foo.com', {noWellknownDat: true})
 
+// specify amount of redirects (default: 7)
```

Comment:
Is it 7 or 6? (Just looking at the constant in index.js)

Reply:
Its 6, fixed in ed33316